extensions-core/kafka-indexing-service/src/main/java/io/druid/indexing/kafka/KafkaIndexTaskClient.java

```diff
@@ -633,7 +677,7 @@ private FullResponseHolder submitRequest(
       }
       catch (Exception e) {
         log.warn(e, "Exception while sending request");
-        throw e;
+        throw new RuntimeException(e);
```

Comment:
>.< another `Exception` catch

Reply:
removed the RuntimeException wrapper here